src/Adapter/MSTest.TestAdapter/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 ﻿#nullable enable
+Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.RunConfigurationSettings.ExecutionApartmentState.get -> System.Threading.ApartmentState?
```

Comment:
do we?

Reply:
I have followed the pattern but that's true we could make it internal. Let me change.